content/lua/concepts/tables/terms/remove/remove.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Remove()'
+Description: 'Removes a value from an array.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.remove()`** function in Lua that removes the value of the specified index and clears the index from the Array (reducing the length of the table by 1).
+
+## Syntax
+
+```lua
+table.remove(tableName,pos)
+```
+
+The `.remove()` function attempts to remove the element at the `pos` position from the table. This causes other elements to shift down to close the space, if necessary.
+This function returns the value that we remove from the table. The `pos` parameter has a default value of `n`, which is the length of the table. This causes the last element of the table to be removed if the `pos` parameter is not provided.
+
+## Example
```

Comment:
I'd like to see the following changes to the example section:
- Move the description/explanation before the code prefaced with something like "In the following example..."
- Have the print statement(s) inside the first code block.
- Have it print the contents of the modified table as well as the removed item.
- Instead of a comment in the code, place the output in it's own ` ```shell` block after the code as follows:

````
This example results in the following output:

```shell
Output from example
```
````